tests/scripts/mbed-build.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
```

Comment:
@RonEld: Normally, the mbed TLS shell scripts should be POSIX compatible. Could you please change the interpreter to `#! /usr/bin/env sh` or `#! /bin/sh`?